GitExtUtils/GitUI/UIExtensions.cs

```diff
@@ -43,5 +45,27 @@ public static bool IsFixedWidth(this Font ft, Graphics g)
 
             return fixedWidth;
         }
+
+        public static string FormatBodyAndNotes(string bodyOrSubject, string? notes)
+        {
+            if (string.IsNullOrWhiteSpace(notes))
+            {
+                return bodyOrSubject;
+            }
+
+            const string notesPrefix = "Notes:";
+            const string indent = "    ";
+            StringBuilder? sb = new StringBuilder((bodyOrSubject?.Length ?? 0) + 2 + notesPrefix.Length + 2 + indent.Length + notes.Length + 1)
+                .AppendLine(bodyOrSubject)
+                .AppendLine(notesPrefix);
+
+            foreach (string? line in notes.Split('\n'))
+            {
+                sb.Append(indent).Append(line).Append('\n');
+            }
+
+            sb.Length--;
```

Comment:
It'd be great to have some comments explaining magic numbers and why we need the decrement on line:67.